e2e/app/page/conceptset-save-modal.ts

```diff
@@ -50,6 +51,10 @@ export default class ConceptsetSaveModal extends Modal {
 
     // Click SAVE button.
     await this.clickButton(LinkText.Save);
+    await Promise.all([
+      waitWhileLoading(this.page),
+      this.waitUntilClose(),
+    ]);
```

Comment:
wait for dialog close and spinner to disappear.

![SearchConcepts_2020-07-13T15_18_47 400Z](https://user-images.githubusercontent.com/35533885/87325321-4d408500-c4ff-11ea-86ec-0c480c079d4a.png)